contexts/did-v1.jsonld

```diff
@@ -42,6 +42,7 @@
       "@id": "dc:created",
       "@type": "xsd:dateTime"
     },
+    "ethereumAddress": "sec:ethereumAddress",
```

Comment:
```suggestion
    "blockchainAccountId": "sec:blockchainAccountId",
```

based on conversations in this thread, and the following definition: https://w3c-ccg.github.io/security-vocab/#blockchainAccountId

this would allow any CAIP account based identifier to be added to a DID Document.

Reply:
@peacekeeper @msporny if this change were merged, would you approve this PR?

Reply:
Yes.

Reply:
Thinking about this more... and playing off of something @csuwildcat said above... and understanding how hard naming is... what about `distributedLedgerAccountId`, `ledgerAccountId`, or `dltAccountId`?

Reply:
hmm, of those three I prefer `ledgerAccountId` ... but still like `blockchainAccountId` best... I chose the name specifically to be respectful of CAIP.

> Chain Agnostic Improvement Proposals (CAIPs) describe standards for blockchain projects that are not specific to a single chain.

> This proposal aims to facilitate specifying accounts for any blockchain extending CAIP-2 blockchain id specification. This is useful for both decentralized applications and wallets to communicate user accounts for multiple chains using string identifiers specific to each chain. Currently wallets are usually designed for each chain and multi-chain wallets use proprietray data structures to differentiate accounts. This proposal aims to standardize these identifiers for accounts to allow interoperability.

> The account id specification will be prefixed with the CAIP-2 blockchain ID and delimited with an at sign (@) commonly used to define addresses.

https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md

I would prefer to keep the language a compromise between readable (no acronym) and respectful of the definition (preserve account_id)....